includes/class-cpt-curator.php

```diff
@@ -414,6 +419,67 @@ public function filter_edit_post_link( $edit_link, $post_id, $context ) {
 	}
 
 	/**
+	 * Modify the curator post trash link to read uncurate and removed the edit inline link.
+	 *
+	 * @param $actions
+	 * @param $post
+	 *
+	 * @return mixed
+	 * @since 0.1.0
+	 */	
+	public function filter_edit_trash_link( $actions, $post ) {
+		
+		if( 'cur-curator' !== $post->post_type ){
+			return;
+		}
+		
+		if ( ! current_user_can( 'delete_post', $post->ID ) ) {
+			return;
+		}
+		
+		if( ! isset( $actions['trash'] ) ) {
+			return;
+		}
+			
+		if ( EMPTY_TRASH_DAYS ) {
```

Comment:
@ayebare before using a constant, be sure to check that it's even defined first or you'll get a warning ... i.e.

```
if ( defined( EMPTY_TRASH_DAYS ) && ! empty( EMPTY_TRASH_DAYS ) ) {
}
```